nwbwidgets/placefield.py

```diff
@@ -426,25 +443,95 @@ def __init__(self, spatial_series: pynwb.behavior.SpatialSeries, **kwargs):
             widgets.VBox([
                 bft_gaussian,
                 bft_spatial_bin_len,
-                dd_unit_select
+                # dd_unit_select
             ]),
             vis2widget(out_fig)
         ]
 
-    def do_1d_rate_map(self, index=0, gaussian_sd=0.0557, spatial_bin_len=0.0168):
+    def do_1d_rate_map(self, gaussian_sd=0.0557, spatial_bin_len=0.0168):
         tmin = min(self.pos_tt)
         tmax = max(self.pos_tt)
 
-        spikes = get_spike_times(self.units, index, [tmin, tmax])
+        index = np.arange(len(self.units))
 
+        spikes = get_spike_times(self.units, index[0], [tmin, tmax])
         xx, occupancy, filtered_firing_rate = compute_linear_firing_rate(
             self.pos, self.pos_tt, spikes, gaussian_sd=gaussian_sd, spatial_bin_len=spatial_bin_len)
 
-        fig, ax = plt.subplots()
+        all_unit_firing_rate = np.zeros([len(self.units), len(xx)])
+        all_unit_firing_rate[0] = filtered_firing_rate
 
-        fig = ax.plot(xx, filtered_firing_rate, '-')
-        ax.set_xlabel('x ({})'.format(self.unit))
-        ax.set_ylabel('firing rate (Hz)')
+        for ind in index[1:]:
+            spikes = get_spike_times(self.units, ind, [tmin, tmax])
+            _, _, all_unit_firing_rate[ind] = compute_linear_firing_rate(
+                self.pos, self.pos_tt, spikes, gaussian_sd=gaussian_sd, spatial_bin_len=spatial_bin_len)
+
+        # npl.set_palette(npl.colors.rainbow)
+        # with npl.FigureManager(show=True, figsize=(8, 8)) as (fig, ax):
+        #     npl.utils.skip_if_no_output(fig)
+        fig, ax = plt.subplots()
+        plot_tuning_curves1D(all_unit_firing_rate, xx, ax=ax, unit_labels=index)
 
+        # fig = ax.plot(xx, filtered_firing_rate, '-')
+        # ax.set_xlabel('x ({})'.format(self.unit))
+        # ax.set_ylabel('firing rate (Hz)')
 
         return fig
+
+def plot_tuning_curves1D(ratemap, bin_pos, ax=None, normalize=False, pad=10, unit_labels=None, fill=True, color=None):
+    """
+    WARNING! This function is not complete, and hence 'private',
+    and may be moved somewhere else later on.
```

Comment:
Can you use numpy docstring here?

e.g. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Can you use a checkbox to control `normalize`?